java/test/jmri/implementation/AbstractConsistManagerTest.java

```diff
@@ -0,0 +1,100 @@
+package jmri.implementation;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+import jmri.Consist;
+import jmri.ConsistListListener;
+import jmri.ConsistListener;
+import jmri.DccLocoAddress;
+import jmri.LocoAddress;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+/**
+ * Tests for AbstractConsistManager
+ */
+public class AbstractConsistManagerTest {
```

Comment:
Could these tests be put into AbstractConsistManagerTestBase instead of creating a new class for the tests?

That will ensure they get run for all consist managers, which are all derived from the AbstractConsistManager implementation.

Reply:
Yes. I have done that now.